### ISSUE TITLE
CO2 similation

### ISSUE THREAD

**Issue by xueshuxiaobai-1234** [senopiano/-GROMACS-MD-The-miscible-process-of-CO2-C8#1]
Did you calculate the CO2 density separately? My calculation results are always incorrect. I would like to ask why.

**Comment by senopiano** [comment]
No. The density of CO2 is not necessary. To calculate the interfacial intension, you need to get the pressure in different directions, and then according to the equation you can obtain the result. And what do you mean by 'incorrect'?



At 2024-12-18 18:47:17, "xueshuxiaobai-1234" ***@***.***> wrote:

Did you calculate the CO2 density separately? My calculation results are always incorrect. I would like to ask why.

**Comment by xueshuxiaobai-1234** [comment]
Thank you very much for your reply, and I am very sorry for replying you late. I want to reproduce the calculation process of CO2 expansion ethanol extraction of rice bran oil, so I first verify the experimental value and literature value of CO2 density calculated by different force fields. However, I used Gromacs software to perform a simple simulation of CO2 at 20Mpa and 308.15K under the EPM2 force field, and calculated the volume of the box according to the density at this temperature and pressure for NVT/NPT calculation. However, in the NVT result analysis, the pressure of the system is 78Mpa, which is very different from the pressure of 20Mpa corresponding to the density at this volume. If it is an NPT calculation, the calculated volume is also very different from the initial setting. I have tried many times and modified the parameters, but the data has always been very different. Can you help me analyze the reason? Thank you very much.Could you help me look at the set parameter file and analyze the problem? Thank you very much



















At 2024-12-18 22:20:07, "Wicky_baby" ***@***.***> wrote:

No. The density of CO2 is not necessary. To calculate the interfacial intension, you need to get the pressure in different directions, and then according to the equation you can obtain the result. And what do you mean by 'incorrect'?



At 2024-12-18 18:47:17, "xueshuxiaobai-1234" ***@***.***> wrote:

Did you calculate the CO2 density separately? My calculation results are always incorrect. I would like to ask why.

**Comment by senopiano** [comment]
分子模拟过程中盒子大小是个虚拟的空间设定，不能根据密度来计算它的体积。在你预平衡的过程中盒子大小其实已经和你原本的设定不一样了。

在 2024-12-20 11:04:59，xueshuxiaobai-1234 ***@***.***> 写道：


Thank you very much for your reply, and I am very sorry for replying you late. I want to reproduce the calculation process of CO2 expansion ethanol extraction of rice bran oil, so I first verify the experimental value and literature value of CO2 density calculated by different force fields. However, I used Gromacs software to perform a simple simulation of CO2 at 20Mpa and 308.15K under the EPM2 force field, and calculated the volume of the box according to the density at this temperature and pressure for NVT/NPT calculation. However, in the NVT result analysis, the pressure of the system is 78Mpa, which is very different from the pressure of 20Mpa corresponding to the density at this volume. If it is an NPT calculation, the calculated volume is also very different from the initial setting. I have tried many times and modified the parameters, but the data has always been very different. Can you help me analyze the reason? Thank you very much.Could you help me look at the set parameter file and analyze the problem? Thank you very much



















At 2024-12-18 22:20:07, "Wicky_baby" ***@***.***> wrote:

No. The density of CO2 is not necessary. To calculate the interfacial intension, you need to get the pressure in different directions, and then according to the equation you can obtain the result. And what do you mean by 'incorrect'?



At 2024-12-18 18:47:17, "xueshuxiaobai-1234" ***@***.***> wrote:

Did you calculate the CO2 density separately? My calculation results are always incorrect. I would like to ask why.

**Comment by xueshuxiaobai-1234** [comment]
请问我应该如何设置盒子呢？Gromacs软件使用EPM2力场计算20MPa380.15K下的1000个CO2分子的密度，尝试了很多次，结果只有400多，但是文献值都是890左右。非常渴望您的帮助。



---- 回复的原邮件 ----
| 发件人 | ***@***.***> |
| 日期 | 2024年12月21日 19:19 |
| 收件人 | ***@***.***> |
| 抄送至 | ***@***.***>***@***.***> |
| 主题 | Re: [senopiano/-GROMACS-MD-The-miscible-process-of-CO2-C8] CO2 similation (Issue #1) |

分子模拟过程中盒子大小是个虚拟的空间设定，不能根据密度来计算它的体积。在你预平衡的过程中盒子大小其实已经和你原本的设定不一样了。

在 2024-12-20 11:04:59，xueshuxiaobai-1234 ***@***.***> 写道：


Thank you very much for your reply, and I am very sorry for replying you late. I want to reproduce the calculation process of CO2 expansion ethanol extraction of rice bran oil, so I first verify the experimental value and literature value of CO2 density calculated by different force fields. However, I used Gromacs software to perform a simple simulation of CO2 at 20Mpa and 308.15K under the EPM2 force field, and calculated the volume of the box according to the density at this temperature and pressure for NVT/NPT calculation. However, in the NVT result analysis, the pressure of the system is 78Mpa, which is very different from the pressure of 20Mpa corresponding to the density at this volume. If it is an NPT calculation, the calculated volume is also very different from the initial setting. I have tried many times and modified the parameters, but the data has always been very different. Can you help me analyze the reason? Thank you very much.Could you help me look at the set parameter file and analyze the problem? Thank you very much



















At 2024-12-18 22:20:07, "Wicky_baby" ***@***.***> wrote:

No. The density of CO2 is not necessary. To calculate the interfacial intension, you need to get the pressure in different directions, and then according to the equation you can obtain the result. And what do you mean by 'incorrect'?



At 2024-12-18 18:47:17, "xueshuxiaobai-1234" ***@***.***> wrote:

Did you calculate the CO2 density separately? My calculation results are always incorrect. I would like to ask why.
